DevelopmentProposals/readme.md

```diff
@@ -0,0 +1,12 @@
+## Overview
+
+This directory contains all the development proposals for OpenSim.
+
+## How to create a proposal
+
+1. Create a new directory under directory ***DevelopmentProposals***.
```

Comment:
I think it was mentioned that Sanbox is going to move to top-level directory in future. I can move this directory to OpenSim.